pygus/gus/models/config.py

```diff
@@ -0,0 +1,117 @@
+import json
+import logging
+from typing import Union, Dict
+
+from ..enums import ProjectSiteType
+
+class ScenarioConfig:
```

Comment:
Below are the objects we accept as input to `Urban`

If `str` is provided, the `.from_file()` methods are used to read that file into the object and continue